filesystems/drs/src/main/scala/cromwell/filesystems/drs/DrsPathBuilderFactory.scala

```diff
@@ -98,13 +98,28 @@ class DrsPathBuilderFactory(globalConfig: Config, instanceConfig: Config, single
     // ONLY use the project id from the User Service Account for requester pays
     val requesterPaysProjectIdOption = options.get("google_project").toOption
 
+    /*
+    `override_preresolve_for_test` is a workflow option to override the default `martha.preresolve` specified in the
+    global config. This is only used for testing purposes.
+     */
+    val preResolve: Boolean =
+      options
+        .getBoolean("override_preresolve_for_test")
```

Comment:
Khalid and I chatted about this in person. I don't love it but I understand there are downsides of other solutions. I would love if we could break the habit of having production code that exists for the sole purpose of an automated test.